Document reputation label

ErikSchierboom opened the pull request:
- Update reputation document to use `x:rep` label
- Document rep for opening issue
- Add backwards compatibility note
- Update rep doc

ErikSchierboom committed:
Update reputation document to use `x:rep` label

ErikSchierboom committed:
Document rep for opening issue

ErikSchierboom committed:
Add backwards compatibility note

ErikSchierboom committed:
Update rep doc

ErikSchierboom commented on using/product/reputation.md:
```diff
@@ -65,21 +65,24 @@ By default, `12` reputation is awarded when a pull request is merged that was op
+| `x:rep/small`   | 5 (~10 mins)  | <ul><li>Fixing a single test case, task or example</li><li>Fixing multiple typos or links in a single file</li><li>Clarifying content by adding a few lines to a file</li></ul>                                                                                                                                                                                     |
+| `x:rep/medium`  | 12 (~30 mins) | <ul><li>Syncing an exercise with problem-specifications (incl. edits)</li><li>Adding one or more test cases from scratch</li><li>Improving multiple files in an exercise</li><li>Adding mentor notes for an exercise from scratch</li><li>Fixing a small bug in a test runner/analyzer/representer</li><li>Adding analyzer comments for a single exericse</li></ul> |
+| `x:rep/large`   | 30 (~2 hrs)   | <ul><li>Adding a new concept or practice exercise</li><li>Adding new concept documentation</li><li>Substantial re-writing of an existing concept or exercise</li><li>Adding new CI scripts or other automation</li></ul>                                                                                                                                            |
+| `x:rep/massive` | 100 (~5 hrs)  | <ul><li>Creating a test-runner, analyzer, representer or generator from scratch</li><li>Major refactors to those tools</li><li>Creating major documentation from scratch (e.g. contribution or testing guides)</li></ul>                                                                                                                                            |
```
I've slightly rounded the estimated time to make it easier to remember.

kotp commented on using/product/reputation.md:
```diff
@@ -65,21 +65,22 @@ By default, `12` reputation is awarded when a pull request is merged that was op
+- The estimated number of time spent should be interpreted as the average time a _maintainer_ would spend on doing the PR.
 - If more than one label is specified, the label with the highest reputation value determines the awarded reputation.
 - If a pull request is still open, no reputation is awarded (yet).
 - If a pull request is closed _without_ merging, no reputation is awarded.
```
This should be "exceptionable".  There can be value in the discussion of code that does not come in, and a historical record of why.  So reputation should be able to be given even if a PR is not accepted.

ErikSchierboom replied:
We have others ways of rewarding reputation for those cases.

kotp replied:
But we clarify with the "(yet)" when it makes little sense to aware while a PR is still open, which is probably something that would be expected, yet, do not clarify that the no reputation for closed pull requests is not absolutely no reputation when warranted?

ErikSchierboom commented:
Note: update the issue section to only mention rep for large/massive issues.

iHiD reviewed (approved):
Merge when you're happy, Erik.

junedev commented:
This is not blocking this particular PR but I wanted to leave a note about two issues I see with the added time values in the reputation table:
* I know there is rounding for nice numbers in place but it looks quite strange now that 10min gives 5 rep but 5h=300min only gives 100 rep instead of 30x10=150 rep.
* Now that the time values are added, the time values and the examples don't seem to fit together anymore. A proper test runner with code extraction from scratch in 5h? A full concept exercise from scratch in 2h with design.md, hints.md etc? Even taking into account extra rep for authoring this does not seem to add up for maintainers that are not 100x devs like Erik or DJ. 

This points to the same root cause imo. The current rep sizes and how they are used overvalue small contributions / undervalue large contributions (despite the fact that we usually need more of the large ones). If you wanted to get the most rep out out of 2h you would do a lot of typo fixes, small grammar improvements etc. Because in reality to create something with a large label like a new concept from scratch on a non-trivial topic gives rep for 2h but takes more like 4h, on complex topics (e.g. JavaScript classes and prototypes), I spent more like 10h to create good content and that does not even include the exercise part that was also in the same PR. My guess is other maintainers have similar stories but I might be wrong.

ErikSchierboom commented:
Yeah, getting these numbers right is very difficult. Maybe we should examine the existing PRs that are large/massive to see what they implemented?

## Large

```shell
{gh search prs --limit 200 --label 'x:size/large' --json 'author,repository,title,url' --merged --jq '.[] | .repository.nameWithOwner + " | " + .title + " | " + .author.login + " | " + .url'; gh search prs --limit 200 --label 'x:rep/large' --json 'author,repository,title,url' --merged --jq '.[] | .repository.nameWithOwner + " | " + .title + " | " + .author.login + " | " + .url' } | sort -t\| -k1 | awk 'BEGIN{print "repo | title | author | url"; print "-- | -- | -- | --"}1'
```

repo | title | author | url
-- | -- | -- | --
exercism/awk-test-runner | AWK test runner | glennj | https://github.com/exercism/awk-test-runner/pull/1
exercism/c | grade-school: align with specification per #503 | wolf99 | https://github.com/exercism/c/pull/555
exercism/common-lisp | Adding Practice Exercise Generator | PaulT89 | https://github.com/exercism/common-lisp/pull/597
exercism/common-lisp | Create format-basics concept & concept exercise. | verdammelt | https://github.com/exercism/common-lisp/pull/537
exercism/csharp | Implement new concept exercise for extension methods | yzAlvin | https://github.com/exercism/csharp/pull/1585
exercism/csharp | Upgrade  projects to .NET 6 | FaisalAfroz | https://github.com/exercism/csharp/pull/1811
exercism/csharp-analyzer | Add analyzer for weighing-machine | Grenkin1988 | https://github.com/exercism/csharp-analyzer/pull/109
exercism/csharp-test-runner | Fix test runner failing for test classes with constructor | ErikSchierboom | https://github.com/exercism/csharp-test-runner/pull/51
exercism/docs | Fix links | ErikSchierboom | https://github.com/exercism/docs/pull/200
exercism/elixir | Bootstrap practice exercises | jiegillet | https://github.com/exercism/elixir/pull/774
exercism/elixir | New concept exercise `city-office` (`docs` and `typespecs`) | angelikatyborska | https://github.com/exercism/elixir/pull/693
exercism/elixir | New concept exercise `dancing-dots` (`use` and `behaviour`) | angelikatyborska | https://github.com/exercism/elixir/pull/1103
exercism/elixir | New concept exercise `top-secret` (concept: `ast`) | angelikatyborska | https://github.com/exercism/elixir/pull/930
exercism/elixir | New concept exercise: `new-passport` (concept: `with`) | jiegillet | https://github.com/exercism/elixir/pull/954
exercism/elixir | New concept exercise: `take-a-number-deluxe` (`genserver`) | angelikatyborska | https://github.com/exercism/elixir/pull/1076
exercism/elixir | New practice exercise `food-chain` | jiegillet | https://github.com/exercism/elixir/pull/780
exercism/elixir | New practice exercise `killer-sudoku-helper` | jiegillet | https://github.com/exercism/elixir/pull/1116
exercism/elixir | New practice exercise `resistor-color-duo` | jiegillet | https://github.com/exercism/elixir/pull/800
exercism/elixir | New practice exercise `resistor-color-trio` | jiegillet | https://github.com/exercism/elixir/pull/801
exercism/elixir | New practice exercise `rest-api` | jiegillet | https://github.com/exercism/elixir/pull/831
exercism/elixir | New practice exercise `sgf-parsing` | jiegillet | https://github.com/exercism/elixir/pull/795
exercism/elixir | New practice exercise `square-root` | jiegillet | https://github.com/exercism/elixir/pull/797
exercism/elixir | New practice exercise `state-of-tic-tac-toe` | jiegillet | https://github.com/exercism/elixir/pull/1115
exercism/elixir | New practice exercise `variable-length-quantity` | jiegillet | https://github.com/exercism/elixir/pull/798
exercism/elixir | Sync: re-write `grade-school` | jiegillet | https://github.com/exercism/elixir/pull/1050
exercism/elixir | Updates to test execution documentation | czrpb | https://github.com/exercism/elixir/pull/940
exercism/elixir | [#722] New practice exercise `yacht` | elibdev | https://github.com/exercism/elixir/pull/760
exercism/elixir | [#723] New practice exercise `zebra-puzzle` | jiegillet | https://github.com/exercism/elixir/pull/753
exercism/elixir | [#724] New practice exercise `rational-numbers` | jiegillet | https://github.com/exercism/elixir/pull/755
exercism/elixir | [#725] New practice exercise `pov` | jiegillet | https://github.com/exercism/elixir/pull/752
exercism/elixir | [#726] New practice exercise `satellite` | jiegillet | https://github.com/exercism/elixir/pull/756
exercism/elixir | [#763] New practice exercise `affine-cipher` | jiegillet | https://github.com/exercism/elixir/pull/773
exercism/elixir | [#764] New practice exercise `book-store` | jiegillet | https://github.com/exercism/elixir/pull/778
exercism/elixir | [#765] New practice exercise `complex-numbers` | jiegillet | https://github.com/exercism/elixir/pull/769
exercism/elixir | [#766] New practice exercise `circular-buffer` | jiegillet | https://github.com/exercism/elixir/pull/775
exercism/elixir | [#767] New practice exercise `darts` | jiegillet | https://github.com/exercism/elixir/pull/777
exercism/elixir | [#768] New practice exercise `go-counting` | jiegillet | https://github.com/exercism/elixir/pull/776
exercism/elixir | [#792] New practice exercise `two-bucket` | jiegillet | https://github.com/exercism/elixir/pull/808
exercism/elixir | [#793] New practice exercise `word-search` | jiegillet | https://github.com/exercism/elixir/pull/804
exercism/elixir-analyzer | 100% test coverage | jiegillet | https://github.com/exercism/elixir-analyzer/pull/246
exercism/elixir-analyzer | Add a common check for `@doc` + `@spec` order | joseemds | https://github.com/exercism/elixir-analyzer/pull/185
exercism/elixir-analyzer | Add a common check for boolean function naming patterns | jiegillet | https://github.com/exercism/elixir-analyzer/pull/180
exercism/elixir-analyzer | Add new `check_source` check to check source code as a string | jiegillet | https://github.com/exercism/elixir-analyzer/pull/189
exercism/elixir-analyzer | Allow reading multiple file solutions and exemploids | jiegillet | https://github.com/exercism/elixir-analyzer/pull/298
exercism/elixir-analyzer | New analyzer function: `assert_call` | neenjaw | https://github.com/exercism/elixir-analyzer/pull/49
exercism/elixir-analyzer | New common check: Helper functions should be private | jiegillet | https://github.com/exercism/elixir-analyzer/pull/207
exercism/elixir-analyzer | [#78] Write a basic square-root analysis | jiegillet | https://github.com/exercism/elixir-analyzer/pull/112
exercism/elixir-representer | Normalize types, typespecs and module attributes | jiegillet | https://github.com/exercism/elixir-representer/pull/52
exercism/elixir-representer | Normalizing multi-component modules and module aliases | jiegillet | https://github.com/exercism/elixir-representer/pull/44
exercism/elm | Add Custom types concept and associated exercise | ceddlyburge | https://github.com/exercism/elm/pull/368
exercism/elm | Add concept exercise for booleans | mpizenberg | https://github.com/exercism/elm/pull/331
exercism/elm | Add list concept and associated exercise | ceddlyburge | https://github.com/exercism/elm/pull/365
exercism/elm | Add record types concept and associated exercise | ceddlyburge | https://github.com/exercism/elm/pull/369
exercism/elm | Add task ID tags to concept exercise tests | jiegillet | https://github.com/exercism/elm/pull/476
exercism/elm | Basics-2 concept and associated exercise | ceddlyburge | https://github.com/exercism/elm/pull/329
exercism/elm | Concept maybe | mpizenberg | https://github.com/exercism/elm/pull/361
exercism/elm | New practice exercise: `complex-numbers` | jiegillet | https://github.com/exercism/elm/pull/482
exercism/elm | Tuples concept and associated exercise | ceddlyburge | https://github.com/exercism/elm/pull/395
exercism/elm-test-runner | Update elm-test-rs to v2.0 and add smoke tests | jiegillet | https://github.com/exercism/elm-test-runner/pull/36
exercism/erlang-test-runner | fully comply to version 2 spec | NobbZ | https://github.com/exercism/erlang-test-runner/pull/22
exercism/fsharp | Implement new Concept Exercise: tuples | Grenkin1988 | https://github.com/exercism/fsharp/pull/1009
exercism/go |  zero_zilch_nada: remove ambiguous reference to empty | sudomateo | https://github.com/exercism/go/pull/1500
exercism/go | Add gomod-sync | andrerfcsantos | https://github.com/exercism/go/pull/1954
exercism/go | Add pointer concept | andrerfcsantos | https://github.com/exercism/go/pull/1724
exercism/go | Add welcome-to-tech-palace exercise | ErikSchierboom | https://github.com/exercism/go/pull/1680
exercism/go | Added new practice exercise: state-of-tic-tac-toe | AbhilashJN | https://github.com/exercism/go/pull/2284
exercism/go | Concept stringers with exercise | norbs57 | https://github.com/exercism/go/pull/2208
exercism/go | Functions concept | bobtfish | https://github.com/exercism/go/pull/1720
exercism/go | Implement new concept with exercise: randomness | norbs57 | https://github.com/exercism/go/pull/2139
exercism/go | New concept First-Class Functions | antklim | https://github.com/exercism/go/pull/2103
exercism/go | Regular expression concept with exercise | norbs57 | https://github.com/exercism/go/pull/2159
exercism/go | Update of testcase generator to latest JSON format | eklatzer | https://github.com/exercism/go/pull/2279
exercism/go | `sorting-room`: Implement exercise to teach about `type-conversion` and `type-assertion` | jmrunkle | https://github.com/exercism/go/pull/1723
exercism/go | bird-watcher: add "for loop" concept | sachinmk27 | https://github.com/exercism/go/pull/1775
exercism/go | interest-is-interesting: add exercise | ErikSchierboom | https://github.com/exercism/go/pull/1740
exercism/go | logs-logs-logs: update concept and exercise content | sudomateo | https://github.com/exercism/go/pull/1707
exercism/go-representer | Implement representer | tehsphinx | https://github.com/exercism/go-representer/pull/15
exercism/groovy-test-runner | Switch to Maven | andreaTP | https://github.com/exercism/groovy-test-runner/pull/18
exercism/java | Implement Connect Exercise | kkyb123 | https://github.com/exercism/java/pull/2027
exercism/java-test-runner | Switch to Maven on infrastructure | andreaTP | https://github.com/exercism/java-test-runner/pull/39
exercism/javascript | :sparkles: Add new exercise : Affine Cipher | TomPradat | https://github.com/exercism/javascript/pull/887
exercism/javascript | Add Concept Exercise: For Loops | junedev | https://github.com/exercism/javascript/pull/1059
exercism/javascript | Add Concept Exercise: Windowing System (Classes) | junedev | https://github.com/exercism/javascript/pull/1476
exercism/javascript | Add concept exercise conditionals | junedev | https://github.com/exercism/javascript/pull/1037
exercism/javascript | Add concept exercise functions | junedev | https://github.com/exercism/javascript/pull/1183
exercism/javascript | Add concept exercise objects | junedev | https://github.com/exercism/javascript/pull/1160
exercism/javascript | Add exercise: resistor-color | SleeplessByte | https://github.com/exercism/javascript/pull/648
exercism/javascript | Add new concept exercise : errors | TomPradat | https://github.com/exercism/javascript/pull/978
exercism/javascript | Add new exercise grep | TomPradat | https://github.com/exercism/javascript/pull/752
exercism/javascript | Add new exercise: resistor-color-trio | SleeplessByte | https://github.com/exercism/javascript/pull/725
exercism/javascript | Add ozans playlist concept exercise | kristinaborn | https://github.com/exercism/javascript/pull/1093
exercism/javascript | Add prerequisites for practice exercises | TomPradat | https://github.com/exercism/javascript/pull/1030
exercism/javascript | Add solution and spec files for elyses-transformative-enchantments | SleeplessByte | https://github.com/exercism/javascript/pull/997
exercism/javascript | Concept exercise: Template string and Ternary | pertrai1 | https://github.com/exercism/javascript/pull/1428
exercism/javascript | Implement D&D Character Exercise | Tyresius92 | https://github.com/exercism/javascript/pull/676
exercism/javascript | Implement resistor-colors exercise | tejasbubane | https://github.com/exercism/javascript/pull/642
exercism/javascript | Improve Concept Exercise Poetry Door Club Policy (strings) | junedev | https://github.com/exercism/javascript/pull/1342
exercism/javascript | Improve Concept Exercise: Type Conversion | junedev | https://github.com/exercism/javascript/pull/1248
exercism/javascript | Improve concept exercise: Null and Undefined | junedev | https://github.com/exercism/javascript/pull/1170
exercism/javascript | Linting overhaul | SleeplessByte | https://github.com/exercism/javascript/pull/1094
exercism/javascript | New Exercise: Promises | slaymance | https://github.com/exercism/javascript/pull/932
exercism/javascript | Port Yacht exercise | ovidiu141 | https://github.com/exercism/javascript/pull/799
exercism/javascript | Sync all exercises | SleeplessByte | https://github.com/exercism/javascript/pull/895
exercism/javascript | Update Fruit Picker -- remove conceptual leaps | neenjaw | https://github.com/exercism/javascript/pull/1725
exercism/javascript | Update concept exercises to pass the linter | SleeplessByte | https://github.com/exercism/javascript/pull/999
exercism/javascript | Update sync and checksum script | SleeplessByte | https://github.com/exercism/javascript/pull/1000
exercism/javascript | [V3] Add Concept Exercise: While and Switch | junedev | https://github.com/exercism/javascript/pull/1073
exercism/javascript | [V3] Annalyn's Infiltration: fix everything around this exercise | SleeplessByte | https://github.com/exercism/javascript/pull/1020
exercism/javascript | [V3] Elyses Enchantments: fix everything around this exercise | SleeplessByte | https://github.com/exercism/javascript/pull/1071
exercism/javascript | [V3] Elyses Looping Enchantments: fix everything around this exercise | SleeplessByte | https://github.com/exercism/javascript/pull/1097
exercism/javascript | [V3] Freelancer Rates: fix everything around this exercise | SleeplessByte | https://github.com/exercism/javascript/pull/1021
exercism/javascript | [V3] Lucian's Lucious Lasagna: fix everything around this exercise | SleeplessByte | https://github.com/exercism/javascript/pull/1018
exercism/javascript | [V3] Poetry Club Door Policy: fix everything around this exercise | SleeplessByte | https://github.com/exercism/javascript/pull/1043
exercism/javascript | [V3] Translation Service: fix everything around this exercise #1071 | SleeplessByte | https://github.com/exercism/javascript/pull/1096
exercism/javascript | add scale-generator exercise | seeksort | https://github.com/exercism/javascript/pull/754
exercism/javascript-test-runner | Allow skipping tests | SleeplessByte | https://github.com/exercism/javascript-test-runner/pull/62
exercism/javascript-test-runner | Initial files | alirezaghey | https://github.com/exercism/javascript-test-runner/pull/1
exercism/javascript-test-runner | Release version 2.0 | SleeplessByte | https://github.com/exercism/javascript-test-runner/pull/26
exercism/javascript-test-runner | Update test runner with console output | SleeplessByte | https://github.com/exercism/javascript-test-runner/pull/29
exercism/kotlin | Update Kotlin and Gradle | incendium | https://github.com/exercism/kotlin/pull/505
exercism/lua-test-runner | lua-test-runner version 2 implemention | imolein | https://github.com/exercism/lua-test-runner/pull/41
exercism/nim-test-runner | Fix per test escaping | ynfle | https://github.com/exercism/nim-test-runner/pull/108
exercism/php | Provide stubs | codedge | https://github.com/exercism/php/pull/357
exercism/python | Add `Sets` Concept Documents | BethanyG | https://github.com/exercism/python/pull/2476
exercism/python | Changed Names of Test Classes to be ExerciseNameTest | BethanyG | https://github.com/exercism/python/pull/2603
exercism/python | Clean up on the List and List-Method Isles | BethanyG | https://github.com/exercism/python/pull/2404
exercism/python | Concept Conditionals Add Files | BethanyG | https://github.com/exercism/python/pull/2430
exercism/python | Concept Exercise Roadmap Updates | BethanyG | https://github.com/exercism/python/pull/2805
exercism/python | Concept and Practice Exercise Mapping | BethanyG | https://github.com/exercism/python/pull/2383
exercism/python | Fix "Making the Grade" ('Loops`) Concept Exercise | BethanyG | https://github.com/exercism/python/pull/2407
exercism/python | Fix error_case test gen to check for 'score' prpty | rneilsen | https://github.com/exercism/python/pull/2840
exercism/python | Fixed Template Error, regenerated tests for Flattened Problem Specs | BethanyG | https://github.com/exercism/python/pull/2421
exercism/python | Gracefully fail with user-friendly error text when unrecognized datac… | cmccandless | https://github.com/exercism/python/pull/2427
exercism/python | Improve test spacing in `chaitanas-colossal-coaster` | pranasziaukas | https://github.com/exercism/python/pull/2659
exercism/python | Issue 2953 (and issue 2745) Revise Track Documentation | BethanyG | https://github.com/exercism/python/pull/2989
exercism/python | Loops spiffying | BethanyG | https://github.com/exercism/python/pull/2406
exercism/python | Meltdown exercise change according to issue #2565 | kbuc | https://github.com/exercism/python/pull/2568
exercism/python | Practice exercise prerequisites & practices tags | BethanyG | https://github.com/exercism/python/pull/2377
exercism/python | Reconstituted V3 Repo PR2949 (https://github.com/exercism/v3/pull/2949) | BethanyG | https://github.com/exercism/python/pull/2420
exercism/python | Redo elyses-enchantments/lists.py as card-games | IsaacG | https://github.com/exercism/python/pull/2405
exercism/python | Sets Concept Exercise | BethanyG | https://github.com/exercism/python/pull/2485
exercism/python | Strings Concept Exercise Re-Write | BethanyG | https://github.com/exercism/python/pull/2433
exercism/python | Update track tooling for v3 structure changes | cmccandless | https://github.com/exercism/python/pull/2311
exercism/python | [Black Jack]: Comparisons Concept and Concept Exercise Finalization | BethanyG | https://github.com/exercism/python/pull/2844
exercism/python | [Cater-Waiter]:  Reduced the Overall Number of Set Tests to 82 | BethanyG | https://github.com/exercism/python/pull/2615
exercism/python | [Inventory Management] Typo &  Language Edits for dicts Concept and Exercise. | BethanyG | https://github.com/exercism/python/pull/2530
exercism/python | [Linting]: Lint practice stub files (1 / ?)  | J08K | https://github.com/exercism/python/pull/2787
exercism/python | [Linting]: Lint practice stub files (2 / ?) | J08K | https://github.com/exercism/python/pull/2789
exercism/python | [Linting]: Lint practice stub files (4 / ?) | J08K | https://github.com/exercism/python/pull/2791
exercism/python | [Linting]: Lint practice stub files (5 / ?) | J08K | https://github.com/exercism/python/pull/2792
exercism/python | [Linting]: Lint practice stub files (8 / 8) | J08K | https://github.com/exercism/python/pull/2809
exercism/python | [New Concept Exercise] : Generators (Plane Tickets) | J08K | https://github.com/exercism/python/pull/2729
exercism/python | [New Concept]: Decorators | mathstrains21 | https://github.com/exercism/python/pull/2838
exercism/python | [New Concept]: Recursion | bobahop | https://github.com/exercism/python/pull/3051
exercism/python | [New Workflow] Bot to automatically reply to new issues | J08K | https://github.com/exercism/python/pull/2684
exercism/python | [New Workflow] PR-Commenter Part 1/2 | J08K | https://github.com/exercism/python/pull/2708
exercism/python | [New Workflow]: Added snippets to the auto-pr-commenter. 2/2 | BethanyG | https://github.com/exercism/python/pull/2707
exercism/python | [New `classes` concept exercise]: ellens-alien-game | PaulT89 | https://github.com/exercism/python/pull/2876
exercism/python | [PR2429 Reconstruction] | BethanyG | https://github.com/exercism/python/pull/2443
exercism/python | [Practice Exercises]: Add Better Error Handling Instructions & Tests for Error Raising Messages (# 6 of 8) | BethanyG | https://github.com/exercism/python/pull/2725
exercism/python | [Practice Exercises]: Add Better Error Handling Instructions & Tests for Error Raising Messages (# 8 of 8) | BethanyG | https://github.com/exercism/python/pull/2734
exercism/python | [Simple Linked List]: Added instruction append and hints | BethanyG | https://github.com/exercism/python/pull/3026
exercism/python | [Update Concept] functions, and complete function arguments | bobahop | https://github.com/exercism/python/pull/3067
exercism/python | [class-inheritance] Concept Documentation | girijakar | https://github.com/exercism/python/pull/2960
exercism/python-analyzer | Issue 20: Exercise Names + A Refactor to Bring Into Spec Compliance | BethanyG | https://github.com/exercism/python-analyzer/pull/22
exercism/ruby-test-runner | Update Ruby to 3.0.2 | katrinleinweber | https://github.com/exercism/ruby-test-runner/pull/48
exercism/scala-test-runner | Compile exercises directly with scalac instead of sbt | jcranky | https://github.com/exercism/scala-test-runner/pull/31
exercism/scheme | New test system | guygastineau | https://github.com/exercism/scheme/pull/262
exercism/swift | Run concept exercises in CI | sanderploegsma | https://github.com/exercism/swift/pull/487
exercism/swift-test-runner | Fix test runner | 2Grey | https://github.com/exercism/swift-test-runner/pull/22
exercism/typescript | Add proofs and stubs for all practice exercises | SleeplessByte | https://github.com/exercism/typescript/pull/443
exercism/typescript | Exercise/resistor color trio | rodmagaldi | https://github.com/exercism/typescript/pull/814
exercism/typescript | New exercise ✨: DnD Character | joshiraez | https://github.com/exercism/typescript/pull/479
exercism/typescript | New exercise ✨: Knapsack  | joshiraez | https://github.com/exercism/typescript/pull/478
exercism/typescript | Pass all the linting | SleeplessByte | https://github.com/exercism/typescript/pull/453
exercism/typescript | Update all GHA workflows | SleeplessByte | https://github.com/exercism/typescript/pull/447
exercism/typescript | Upgrade sync script | SleeplessByte | https://github.com/exercism/typescript/pull/444
exercism/typescript-analyzer | Update for v3 spec | SleeplessByte | https://github.com/exercism/typescript-analyzer/pull/18
exercism/website | Add progress svg graph to website journey and track show page. | neenjaw | https://github.com/exercism/website/pull/1181
exercism/website | Update progress graph line algorithm | neenjaw | https://github.com/exercism/website/pull/1239
exercism/x86-64-assembly | add collatz conjecture exercise | sudhackar | https://github.com/exercism/x86-64-assembly/pull/146

## Massive

```shell
{gh search prs --limit 200 --label 'x:size/massive' --json 'author,repository,title,url' --merged --jq '.[] | .repository.nameWithOwner + " | " + .title + " | " + .author.login + " | " + .url'; gh search prs --limit 200 --label 'x:rep/massive' --json 'author,repository,title,url' --merged --jq '.[] | .repository.nameWithOwner + " | " + .title + " | " + .author.login + " | " + .url' } | sort -t\| -k1 | awk 'BEGIN{print "repo | title | author | url"; print "-- | -- | -- | --"}1'
```

repo | title | author | url
-- | -- | -- | --
exercism/elm | Create a practice exercise generator | jiegillet | https://github.com/exercism/elm/pull/480
exercism/elm-analyzer | Add docker integration, CI, and internal `elm-review` | jiegillet | https://github.com/exercism/elm-analyzer/pull/27
exercism/elm-analyzer | Proposal for Elm Analyzer | jiegillet | https://github.com/exercism/elm-analyzer/pull/12
exercism/emacs-lisp-test-runner | Upgrade to version 2 spec | yshym | https://github.com/exercism/emacs-lisp-test-runner/pull/31
exercism/eslint-config-javascript | Bump dependencies and add prettier | SleeplessByte | https://github.com/exercism/eslint-config-javascript/pull/7
exercism/eslint-config-javascript | Remove guard-for-in | SleeplessByte | https://github.com/exercism/eslint-config-javascript/pull/1
exercism/eslint-config-tooling | Add prettier | SleeplessByte | https://github.com/exercism/eslint-config-tooling/pull/9
exercism/eslint-config-typescript | Add prettier | SleeplessByte | https://github.com/exercism/eslint-config-typescript/pull/6
exercism/haxe | Add all concepts and fix CI | midyHamdoun | https://github.com/exercism/haxe/pull/199
exercism/javascript-analyzer | Design and apply a proper interface and structure | SleeplessByte | https://github.com/exercism/javascript-analyzer/pull/18
exercism/javascript-analyzer | Upgrade and use @exercism/static-analysis | SleeplessByte | https://github.com/exercism/javascript-analyzer/pull/75
exercism/javascript-lib-static-analysis | Add badges to readme | SleeplessByte | https://github.com/exercism/javascript-lib-static-analysis/pull/30
exercism/javascript-lib-static-analysis | Create LICENSE | SleeplessByte | https://github.com/exercism/javascript-lib-static-analysis/pull/22
exercism/javascript-representer | Add prettier | SleeplessByte | https://github.com/exercism/javascript-representer/pull/30
exercism/javascript-test-runner | Release version 2.0 | SleeplessByte | https://github.com/exercism/javascript-test-runner/pull/26
exercism/javascript-test-runner | Rewrite the test runner to work with Jest 27 | SleeplessByte | https://github.com/exercism/javascript-test-runner/pull/68
exercism/javascript-test-runner | Update interface and use @exercism/static-analysis | SleeplessByte | https://github.com/exercism/javascript-test-runner/pull/31
exercism/julia-test-runner | Collapse testsets | cmcaine | https://github.com/exercism/julia-test-runner/pull/77
exercism/nim-representer | Feature: upgrade macro evaluation strategy to use `nimscripter` | ynfle | https://github.com/exercism/nim-representer/pull/37
exercism/python | New contributing guidelines and README | BethanyG | https://github.com/exercism/python/pull/2660
exercism/python | [Maintenance]: Sync Practice Exercises from Problem Specification Repo | BethanyG | https://github.com/exercism/python/pull/2914
exercism/python | [Python Track Syllabus]:  Track Re-Ordering | BethanyG | https://github.com/exercism/python/pull/3057
exercism/python | [TESTING.md] Enhance TESTING docs | J08K | https://github.com/exercism/python/pull/2611
exercism/scala | Bump Scala to version `2.13` and scalatest to version `3.2.10` | CJSmith-0141 | https://github.com/exercism/scala/pull/712
exercism/tracks-maintenance-dashboard | Add CI (install, build, test) | SleeplessByte | https://github.com/exercism/tracks-maintenance-dashboard/pull/1
exercism/tracks-maintenance-dashboard | Make it work with v3 repos | SleeplessByte | https://github.com/exercism/tracks-maintenance-dashboard/pull/59
exercism/typescript | Migrate to Jest 17, ESM, and config packages | SleeplessByte | https://github.com/exercism/typescript/pull/944
exercism/typescript-analyzer | Fix build and prepare step | SleeplessByte | https://github.com/exercism/typescript-analyzer/pull/3
exercism/typescript-analyzer | Sync with javascript analyzer | SleeplessByte | https://github.com/exercism/typescript-analyzer/pull/2
exercism/typescript-representer | Initial release based on the javascript-representer | SleeplessByte | https://github.com/exercism/typescript-representer/pull/1
exercism/typescript-test-runner | Migrate to Jest 17, and add support for task ID | SleeplessByte | https://github.com/exercism/typescript-test-runner/pull/38
exercism/typescript-test-runner | 👶🏽 hello there my sweet runner | SleeplessByte | https://github.com/exercism/typescript-test-runner/pull/3
exercism/v3-dashboard | Bump versions in CI to Node LTS | SleeplessByte | https://github.com/exercism/v3-dashboard/pull/116
exercism/vbnet-test-runner | Add initial version of test runner | ErikSchierboom | https://github.com/exercism/vbnet-test-runner/pull/5

ErikSchierboom committed:
Remove time estimates

ErikSchierboom commented:
I've merged this PR as I want to get the docs updated. We should however continue discussing what the proper reputation values should be.

junedev commented:
Could you maybe create a new issue (not sure where exactly) and paste your table (and suggested time values?) in there? Then we can ask some other people to give input as well. I think looking at those examples is a good idea but I don't think this PR is the right place for the discussion.

ErikSchierboom commented:
Issue opened: https://github.com/exercism/exercism/issues/6440

SleeplessByte commented:
(just here today thanks for thinking I'm a 100x dev)

I agree with your assessment and I'll try to provide some hour estimates in the linked issue. I spend more than 5 hours on many of the massive prs and that's... Even with being a NX dev.
